test/realm_object_test.dart

```diff
@@ -20,7 +20,8 @@
 
 import 'dart:io';
 import 'package:test/test.dart' hide test, throws;
-import '../lib/realm.dart';
+import '../lib/realm.dart' hide RealmObject;
+import '../lib/src/realm_object.dart' show RealmObject, RealmObjectInternal;
```

Comment:
We're not exposing `toJson` publicly because it's not a sensible json representation of the object, but I need it for the tests because that's the only way to verify that what Core sees is the right thing.

Reply:
these are not needed now